src/jdk.jfr/share/classes/jdk/jfr/internal/dcmd/DCmdDump.java

```diff
@@ -227,7 +227,8 @@ private PlatformRecording newSnapShot(PlatformRecorder recorder, Recording recor
                                  For example, "-12h", "-15m" or "-30s"
 
                  filename        (Optional) Name of the file to which the flight recording data is
-                                 dumped. If no filename is given, a filename is generated from the PID
+                                 dumped. The string '%%p' in the filename will be replaced with the
```

Comment:
Should this be `%p` and not `%%p`?

Reply:
This method invokes String::formatted of this String, so we need '%%p' here.

https://github.com/openjdk/jdk/pull/4550/files/f3fed5ab7e00986cf685287f6654215145f6bb40#diff-48d2f4fec466ce18084c7fe1ad30c5a842f93ab85713efcf6c6108b0574383c5L272

Reply:
🆗